_posts/2022-03-07-gsoc2022.md

```diff
@@ -0,0 +1,126 @@
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-1-bead-and-ring-groups)                                         | medium     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-matta |
+| 2       | [Type hinting]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-2-type-hinting)                              | medium      | 175 hours   | Add type hints to the MDAnalysis library                                                          | Python        | @IAlibay, @jbarnoud |
+| 3       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-3-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope  |
+| 4       | [Benchmarking and performance optimization]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-4-benchmarking-and-performance-optimization)                  | medium     | 175 hours   |  write benchmarks for automated performance analysis and address performance bottlenecks | Python                      | @hmacdope, @orbeckst, @jbarnoud  |
+| 5       | [Context-aware guessers]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-5-context-aware-guessers) | medium | 350 hours   | Extend how the library guesses properties such as bonds, masses or atom symbols; and write guessers that know about the context of the system (database of origin, force field...) | Python, Molecular modelling | @jbarnoud @micaela-matta @IAlibay |
+
+# Information for Students
+
+You must meet our [own requirements] if you want to be a student with MDAnalysis
+this year (read all the docs behind these links!). You must also meet the
+[eligibility criteria]. Our [GSoC FAQ][] collects common questions
+from students.
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
```

Comment:
```suggestion
Introduce yourself on the [mailing list]. Tell us your github handle, what you plan to work
```
Or something along these lines, linking folks from their email to their PR is always difficult.